api/src/main/java/edu/cornell/mannlib/vitro/webapp/rdfservice/impl/ChangeSetImpl.java

```diff
@@ -48,28 +48,50 @@ public List<ModelChange> getModelChanges() {
 	    	return modelChanges;
 	}
 
+	@Deprecated
 	@Override
 	public void addAddition(InputStream model, RDFService.ModelSerializationFormat format, String graphURI) {
```

Comment:
This is still used in some classes (RDFServiceGraph, RDFServiceImpl, RestoreModelsAction, RDFServiceBulkUpdater). Does it mean that changes are not audit properly in those cases? The same question is for the deprecated addRemoval method

Reply:
Yes